test/dummy/app/views/home/index.html

```diff
@@ -1,8 +1,29 @@
-<h1>Hello!</h1>
-<%= copilot_text 'foobar' do %>
-  I am editable
+<ol>
+  <li>Home</li>
+  <li><%= link_to 'About', about_path %></li>
```

Comment:
not exactly.  you need an html element bc we're using the contenteditable attribute.  However, you could use the block syntax for link_to and use copilot_text there and that would work:

```
<%= link_to about_path do %>
  <%= copilot_text 'foo.link', 'About' %>
<% end %>
```